### PR TITLE
[FIX] sale: fix mail template

### PR DESCRIPTION
**Pull request by fdamhaut**
Fix mail template for better intergration in sale subscription. Improve english too.

task-id : 3366756

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by fdamhaut** [comment]
@odoo/rd-sales Would you mind reviewing ?
Just a small improvement of english + fix for the template as if compare return 0 would mean they will also confirm the SO

File: addons/sale/data/mail_template_data.xml [hunk 1]
```diff
@@ -262,20 +262,22 @@
...
-                to pay.
+                will remain to be paid.
             </t>
         </t>
         <t t-else="">
             has been confirmed.
-            <br/>
-            It remains <span style="font-weight:bold;" t-out="format_amount(object.amount_total - object.amount_paid, object.currency_id) or ''">$ 10.00</span>
-            to be paid.
+            <t t-if="not object.currency_id.compare_amounts(object.amount_paid + transaction_sudo.amount, object.amount_total) &lt; 0">
```
Comment (vava-odoo) [inline addons/sale/data/mail_template_data.xml:276]: Why is this change for?

**Review by vava-odoo — COMMENTED** [review]
Small question about a change, otherwise it looks good to me (English + other change). Thanks!

File: addons/sale/data/mail_template_data.xml (hunk 1 above)
Comment (fdamhaut) [reply addons/sale/data/mail_template_data.xml:276]: If nothing remains to be paid, There is no point to say "0$ remains to be paid" (+ It avoid issue if the payment have exceeded the amount to be paid and say "-X$ remains to be paid") Tho if you believe it's a plus, I can remove that part.

Comment (vava-odoo) [reply addons/sale/data/mail_template_data.xml:276]: I agree it's a plus. I'd rather avoid the negation though 
```suggestion
            <t t-if="object.currency_id.compare_amounts(object.amount_paid + transaction_sudo.amount, object.amount_total) >= 0">
```

**Review by vava-odoo — APPROVED** [review]

**Comment by arj-odoo** [comment]
@robodoo r+

**Pull request closed without merging**